test/puzzle-test.js

```diff
@@ -0,0 +1,24 @@
+import chai from 'chai';
+import Game from '../src/Game';
+import data from '../src/data';
+import Player from '../src/Player';
+import Round from '../src/Round';
+import Puzzle from '../src/Puzzle';
+
+const expect = chai.expect;
+
+describe('Puzzle', function() {
+  let puzzle;
+  let game;
+
+  beforeEach(function() {
```

Comment:
Take some time to cleanup the indentation here. You don't want someone peering into a code file and seeing indentation that isn't correct. At first glance, they might think attention to detail could be lacking.